packages/react/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -25,6 +26,7 @@ export interface ToolbarItemProps
     ContentComponentProps {
   /**
    * Accessibility behavior if overridden by the user.
+   * @default toolbarItemBehavior
```

Comment:
wasn't this supposed to have the same value as the one in `defaultProps`?

Reply:
yup, you're right